docs/UserGuide.md

```diff
@@ -3,56 +3,66 @@ layout: page
 title: User Guide
 ---
 
-Financial Assurance Revolutionary Telemarketer (F.A.R.T) is a **desktop app for managing contacts just for you Financial Advisors, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, F.A.R.T can get your contact management tasks done faster than traditional GUI apps.
+Financial Assurance Revolutionary Telemarketer (F.A.R.T) is a**desktop app for managing contacts just for you Financial Advisors, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, F.A.R.T can get your contact management tasks done faster than traditional GUI apps.
 
 * Table of Contents
 {:toc}
 
+--------------------------------------------------------------------------------------------------------------------
+ ## Glossary
+
+* Command Line Interface (CLI): All instructions are done via typing into a textbox at the bottom of your screen
+* Command Terminal:
+    * If you are a Windows user, press the Windows key and in the search bar, type in Powershell. Click on it and that will be your Command Terminal
+    * For Mac/Linux users, look for Terminal under applications and that will be your Command Terminal
+* Graphical User Interface (GUI): A visual platform for you to interact with the F.A.R.T. book. i.e., the window that opens upon starting the book.
+* Parameter: Something that must be typed into the Command Line for the command to be properly recognised
+* Index: The number found beside the client's name in the F.A.R.T book
+
 --------------------------------------------------------------------------------------------------------------------
 
-## Quick start
+
+ ## Quick start
 
 1. Ensure you have Java `17` or above installed in your Computer.
    1. You can check which version of Java you are currently running by entering `java -version` into a command terminal.
-   2. Should you require help, [here](https://www.oracle.com/java/technologies/downloads/#java17) is the download link to Java '17'
-   3. After accessing the website, please choose the right download link for your operating system (Linux, macOS or Windows)
+   1. Should you require help, [here](https://www.oracle.com/java/technologies/downloads/#java17) is the download link to Java 17
+   1. After accessing the website, please choose the right download link for your operating system (Linux, macOS or Windows)
 
-2. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-F14b-4/tp/releases/tag/v1.4).
+1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-F14b-4/tp/releases/tag/v1.4).
```

Comment:
the latest .jar file is v1.6, unfortunately will have to change this right before submitting final version